node/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@stroeer/tapir",
+  "name": "@stroeer/tapir-v1",
```

Comment:
sage voraus dass wir für immer auf der v1  arbeiten werden und wir diesen Suffix irgendwann verfluchen werden.  
My two cents.

Reply:
ich dachte die Versionsnummer verschwindet dann aus dem package. So mutch duplication :(

```ts
import { ArticleServiceClient } from '@stroeer/tapir/v1/article/service_grpc_pb';
// vs.
import { ArticleServiceClient } from '@stroeer/tapir-v1/stroeer/article/v1/service_grpc_pb';
```

IMO:

```ts
import { ArticleServiceClient } from '@stroeer/tapir-v1/article/service_grpc_pb';
// or
import { ArticleServiceClient } from '@stroeer/tapir/v1/article/service_grpc_pb';
```

Reply:
IMO we should follow the Google Naming guidelines  here: https://cloud.google.com/apis/design/naming_convention#package_names

All good examples I have seen (e.g [google](https://github.com/googleapis/googleapis) or [envoy](https://github.com/envoyproxy/data-plane-api/tree/master/envoy/api/v2)) do muse version suffixes in their proto packages. 

If we use versions in proto, we should include that information in the client libraries as well. From [buf](https://buf.build/docs/lint-checkers):

> One of the core promises of Protobuf API development is to never have breaking changes in your APIs, and Buf helps enforce this through the breaking change detector. However, there are scenarios where you do want to properly version your API. Instead of making changes, the proper method to do so is to make a completely new Protobuf package that is a copy of your existing Protobuf package, serve both packages server-side, and manually migrate your callers. This checker enforces that all packages have a version attached so that it is clear when a package represents a new version.

Nevertheless we could agree on not using a version at all and see how far we get (probably good enough).